.github/workflows/docker-build.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       - name: Check out the repo
         uses: actions/checkout@v3
```

Comment:
The dedicated checkout action is probably no longer needed - `docker/build-push-action` uses git context by default https://github.com/marketplace/actions/build-and-push-docker-images#git-context

Reply:
Oh yep, good catch. Will take this for a test run shortly.

Reply:
I tried it without the checkout action but it doesn't work because we sync submodules in the Dockerfile